handleForms.js

```diff
@@ -0,0 +1,38 @@
+const form = document.querySelector('.form-list');
+const email = form.elements[1];
+
+function error(input, message) {
+  const error = input.previousElementSibling;
+  error.innerText = message;
+  return false;
+}
+
+function success(input) {
+  const error = input.previousElementSibling;
+  error.innerText = '';
+  return true;
+}
+
+function requireValue(input, message) {
+  return input.value.trim() === '' ? error(input, message) : success(input);
+}
+
+function validateEmail(input) {
+  const re = /^[a-z]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+
+  return re.test(input.value.trim())
+    ? success(input)
+    : error(input, 'Invalid email format');
+}
```

Comment:
- [ ] I kindly invite you to show the message error closer to submit button, preferably at the top, this could help the user knows about this error when tries to click the button. 🐶 
![image](https://user-images.githubusercontent.com/20563252/123095136-56b2bc80-d3f3-11eb-99f7-ad82d6600c18.png)

Reply:
Thanks for your helpful reviews.

Reply:
I changed the place of alert text.